dbt/include/clickhouse/macros/materializations/incremental.sql

```diff
@@ -72,6 +76,12 @@
       {% call statement('main') %}
         {{ clickhouse__insert_into(target_relation, sql) }}
       {% endcall %}
+    {% elif incremental_strategy == 'insert_replace' %}#}
+      {%- set partition_by = config.get('partition_by') -%}
```

Comment:
Maybe also better to raise exception if `partition_by` is empty

Reply:
Does not `.get` raise an exception in Jinja if the key is missing? Or does it have regular Python's dict behaviour?